core/src/main/scala/com/databricks/labs/remorph/intermediate/extensions.scala

```diff
@@ -234,11 +234,25 @@ case object WEEK_INTERVAL extends KnownIntervalType
 case object MONTH_INTERVAL extends KnownIntervalType
 case object YEAR_INTERVAL extends KnownIntervalType
 
-// TSQL - For translation purposes, we cannot use teh standard Catalyst CalendarInterval as it is not
+// TSQL - For translation purposes, we cannot use the standard Catalyst CalendarInterval as it is not
 // meant for code generation and converts everything to microseconds. It is much easier to use an extension
 // to the AST to represent the interval as it is required in TSQL, where we need to know if we were dealing with
 // MONTHS, HOURS, etc.
 case class KnownInterval(value: Expression, iType: KnownIntervalType) extends Expression {
   override def children: Seq[Expression] = Seq(value)
   override def dataType: DataType = UnresolvedType
 }
+
+// SNOWFLAKE - Common Table Expression may be of the form:
+// WITH
+//    a AS (1),
+//    b AS (2)
+// SELECT ...
+// CTEs like `a AS (1)` above act as columns of an anonymous table with a single row.
+// In AstBuilding phase, we translate those as InlineColumnExpression. Later, in Optimizing phase, we'll combine all
+// such expressions in a single table declaration (using VALUES).
+case class InlineColumnExpression(columnName: Id, value: Expression) extends LogicalPlan {
```

Comment:
```suggestion
private[snowflake] case class InlineColumnExpression(columnName: Id, value: Expression) extends LogicalPlan {
```
and more it to the snowflake folder.